data/sidebar.yml

```diff
@@ -16,9 +16,37 @@ solutions:
 
 references:
   breakage:
-    - https://github.com/webcompat/web-bugs/issues/11622
-    - https://github.com/webcompat/web-bugs/issues/55685
-    - https://github.com/webcompat/web-bugs/issues/67848
-    - https://github.com/webcompat/web-bugs/issues/106830
+    - url: https://github.com/webcompat/web-bugs/issues/11622
+      site: http://www.election.gov.np
+      platform:
+        - all
+      intervention: https://github.com/mozilla-extensions/webcompat-addon/blob/4a3094f52d561925620ac441d5c4423766ec5c29/src/webextension/injections/js/bug1472081-election.gov.np-window.sidebar-shim.js
+      impact: feature_broken
+      affects_users: all
+      resolution: site_changed
+    - url: https://github.com/webcompat/web-bugs/issues/55685
+      site:  http://www.susu09.com/forum.php
```

Comment:
Wonder if we want to store just the domain name here? Though probably not a big deal as we could parse and extract it to determine site rank on the computing stage :)

Reply:
Yes, I think for the tranco list the hard part is actually knowing what the eTLD+1 is, since I think that's what it's storing (e.g. we don't want to lookup `drive.google.com` but just `google.com`). So I think putting the whole URL is better because a) it's easier to copy and b) it preserves information in case we can do more accurate lookups in the future, without causing any additional difficulty because we still need to parse it and do a public suffix list lookup.

Reply:
> Yes, I think for the tranco list the hard part is actually knowing what the eTLD+1 is, since I think that's what it's storing (e.g. we don't want to lookup drive.google.com but just google.com). 

That made me think that we have logic for this on webcompat.com (it extracts url from the issue body, checks first level domain and then less level domain, gets the rank from Tranco db or Alexa db) https://github.com/webcompat/webcompat.com/blob/main/webcompat/webhooks/helpers.py#L122. We use it for adding priority labels, but I was planning on changing it to add numerical rank in https://github.com/webcompat/webcompat.com/issues/3707.

Maybe I could add an API as well, which we can call to get the rank when computing the overall impact? We have Tranco data and can fetch more per country data from Alexa (while it's still accessible).

Reply:
Yeah, an API that takes a URL or a (full) domain and returns the site rank sounds amazing. I was looking at implementing it, but it makes sense that you already have it for webcompat.com and calling an API is easier than writing all the logic :)

Reply:
Alright, I'll look into it then!

Reply:
Although, thinking about it, for an API we would presumably want to have rate limits to avoid abuse, and that could make the client code more complex than just downloading the tranco list and matching each part of the domain from the end. So if you don't get to the API part don't worry.